code/game/objects/random/random.dm

```diff
@@ -274,7 +274,7 @@
 				prob(1);/obj/item/clothing/suit/storage/vest/heavy/merc,
 				prob(1);/obj/item/weapon/beartrap,
 				prob(1);/obj/item/weapon/handcuffs/fuzzy,
-				prob(1);/obj/item/weapon/handcuffs/legcuffs,
+				prob(1);/obj/item/weapon/handcuffs/legcuffs/fuzzy,
```

Comment:
//VOREStation Edit

Reply:
There is an already existing edit comment above this whole list indicating that it is changed.

Reply:
Oof.